proto/regen/ecocredit/basket/v1/types.proto

```diff
@@ -15,3 +15,9 @@ message BasketCredit {
   //  credit type for this batch.
   string amount = 2;
 }
+
+// BasketCredits represents the slice of BasketCredit objects.
+message BasketCredits {
+  // credits is the list of basket credits
+  repeated BasketCredit credits = 1;
+}
```

Comment:
I believe we want changes to proto definitions merged into master and backported to the release branch. We will need to add this addition to master as well, correct? @aaronc @technicallyty @robert-zaremba

Reply:
I think we can do either way (unless there is a change in state.proto). Those backports needs to be done manually.